dockerfiles/dev/Dockerfile

```diff
@@ -26,25 +26,26 @@
 #
 
 # build using the TerraFERMA development environment
-FROM terraferma/dev-env:fenics-2019.1.0
+FROM terraferma/dev-env:fenics-2019.1.0-focal
```

Comment:
Can similarly be renamed to just fenics-2019.1.0

Reply:
ditto

Reply:
ditto ditto